src/lib/protocol/encryption/README.md

```
@@ -0,0 +1,135 @@
+[version=0] [numClients]
+per client group:
+  [clientId] [numEntries]
+  [clocks as IntDiffOptRle bytes (varUint8Array)]
+  [contentRefs as UintOptRle bytes (varUint8Array)]
+  [data lengths as UintOptRle bytes (varUint8Array)]
+  [concatenated data (raw bytes, length = sum of data lengths)]
+```
+
+## Wire format (ContentEncryptedPayload)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)